File: shoring/flywheel/src/test/java/com/panopset/tests/flywheel/ExecuteTest.java
package com.panopset.tests.flywheel;

import java.io.IOException;
import org.junit.jupiter.api.Test;

public class ExecuteTest  {

  @Test
  void testExecuteCommand() throws IOException {
    new SimpleTest().comparisonTest("executeTest.txt", "executeTest.txt",
        "executeTestExpected.txt");
  }
}

File: code/shoring/flywheel/src/test/kotlin/com/panopset/tests/flywheel/ExecuteTest.kt
package com.panopset.tests.flywheel

import org.junit.jupiter.api.Test

class ExecuteTest {
    @Test
    fun testExecuteCommand() {
        SimpleTest().comparisonTest(
            "executeTest.txt", "executeTest.txt",
            "executeTestExpected.txt"
        )
    }
}
